### ISSUE TITLE
'Not' colour

### ISSUE THREAD

**Issue by TodePond** [TodePond/CellPond#198]
It would be helpful to be able to say "any colour except ___" somehow!

**Comment by 100xCode** [comment]
Yes also this "not" shape should be able to refer to a class of squares as well.

**Comment by TeraByte-Official** [comment]
I think it should be a plus shape.

**Comment by TodePond** [comment]
![Screenshot_20221117-073659](https://user-images.githubusercontent.com/15892272/202394153-d6a609ea-f7e4-4000-b16b-72d741813649.png)

**Comment by TeraByte-Official** [comment]
Why not?
